cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationOperations.java

```diff
@@ -19,20 +34,199 @@ public ApplicationOperations(DefaultCloudFoundryOperations cloudFoundryOperation
     }
 
     public List<ApplicationBean> getAll() {
-        List<ApplicationSummary> applications = this.cloudFoundryOperations.applications().list().collectList().block();
+        List<ApplicationSummary> applications = this.cloudFoundryOperations
+                .applications()
+                .list()
+                .collectList()
+                .block();
 
         // create a list of special bean data objects, as the summaries cannot be serialized directly
         List<ApplicationBean> beans = new ArrayList<>();
         for (ApplicationSummary summary : applications) {
-            GetApplicationManifestRequest manifestRequest = GetApplicationManifestRequest.builder()
-                    .name(summary.getName()).build();
-            ApplicationManifest manifest = this.cloudFoundryOperations.applications()
-                    .getApplicationManifest(manifestRequest).block();
+            ApplicationManifest manifest = getApplicationManifest(summary);
 
             beans.add(new ApplicationBean(manifest));
         }
 
         return beans;
     }
 
+    private ApplicationManifest getApplicationManifest(ApplicationSummary applicationSummary) {
+        return this.cloudFoundryOperations
+                .applications()
+                .getApplicationManifest(GetApplicationManifestRequest
+                        .builder()
+                        .name(applicationSummary.getName())
+                        .build())
+                .block();
+    }
+
+    /**
+     *
+     *  Pushes the app to the cloud foundry instance specified within the cloud foundry operations instance
+     *
+     * @param bean  application bean that holds the configuration settings to deploy the app
+     *              to the cloud foundry instance
+     * @param shouldStart   if the app should not start after being created
```

Comment:
I am confused by the naming of shouldStart  variable.

Reply:
modified the documentation to shouldStart to reflect it's proper meaning